examples/skills/pptx/scripts/office/validators/base.py

```
@@ -0,0 +1,854 @@
+                + "Broken references MUST be fixed, "
+                + "and unreferenced files MUST be referenced or removed."
+            )
+            return False
+        else:
+            if self.verbose:
+                print(
+                    "PASSED - All references are valid and all files are properly referenced"
+                )
+            return True
+
+    def validate_all_relationship_ids(self):
```

Ruff (0.15.13)

[warning] Too many branches (18 > 12)

(PLR0912)